jax/__init__.py

```diff
@@ -247,6 +247,24 @@
       ' disable jax.config.jax_jit_pjit_api_merge.'
   )
 
+import sys as _sys
+if _sys.version_info < (3, 9):
+  from warnings import warn as _warn
+  import jax._src.cloud_tpu_init as _cloud_tpu_init
+  msg = ("jax will drop Python 3.8 support in an upcoming future release "
+         "(see https://jax.readthedocs.io/en/latest/deprecation.html). "
+         "Please upgrade your Python version to 3.9 or higher. "
+         f"Current version: {_sys.version.split(' ')[0]}")
+  if _cloud_tpu_init.running_in_cloud_tpu_vm or True:
```

Comment:
Why the `or True`?

Reply:
Oh whoops meant to remove this, good catch. Was testing the message on a non-TPU VM.